### PR TITLE
Solves exercise ex10

### PR DESCRIPTION
**Pull request by PedroPapa1**
10. **Repeat Integer Display**  
    Write an algorithm to read an integer and display it on the screen 10 times.

### CODE REVIEW

File: ex10/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+console.log("Insert a integer number:");
+const num = Number(prompt(">"));
+
+function repeat() {
+    for (let i = 0; i < 10; i++)
+        console.log(num);
+}
+
+console.log(repeat());
```
Comment (emiliosheinz) [inline ex10/index.js]: Why are you calling `console.log` on a function that does not return anything?

File: ex10/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert a integer number:");
+const num = Number(prompt(">"));
+
+function repeat() {
+    for (let i = 0; i < 10; i++)
```
Comment (a-ahandani) [inline ex10/index.js]: Although the code works as you wrote it, I highly recommend using curly braces {} on `for` loops. It prevents many mistakes in the future and makes the code easier to read.
```js
for (let i = 0; i < 10; i++) {
  // your code
}
```

File: ex10/index.js (hunk 1 above)
Comment (PedroPapa1) [reply ex10/index.js]: I hadn't thought about that, fixed.

**Review by emiliosheinz — APPROVED** [review]

File: ex10/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert a integer number:");
+const num = parseInt(prompt(">"));
+
+function repeatNumbers() {
+    for (let i = 0; i < 10; i++) {
```
Comment (VictorComette) [inline ex10/index.js]: magic number

**Review by VictorComette — APPROVED** [review]

**Pull request merged**